site/smart-accounts/accounts/modular-account.md

```diff
@@ -18,33 +18,46 @@ head:
       content: Coming soon! Follow this guide to use Modular Accounts with Account Kit, a vertically integrated stack for building apps that support ERC-4337.
```

Comment:
Hmm this files seems out of scope of light account? Why include this removal in this PR? Am I missing context?